### PR TITLE
Added cifar10 cnn source - Mostly same as example code :(

### PR DESCRIPTION
**Pull request by ryonakai**

### CODE REVIEW

File: nakai/alexnet.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+# convert class vectors to binary class matrices
+Y_train = np_utils.to_categorical(y_train, nb_classes)
+Y_test = np_utils.to_categorical(y_test, nb_classes)
+
+model = Sequential()
+
+model.add(Convolution2D(3, 11, 11, border_mode='same', input_shape=(3,32,32)))
+model.add(BatchNormalization((48,226,226)))
+model.add(Convolution2D(32, 3, 3))
```
Comment (ryonakai) [inline nakai/alexnet.py:48]: http://keras.io/layers/normalization/ 
mode => 1 にする必要あり
